### PR TITLE
Kimi k2.5 CI fix

### PR DESCRIPTION
**Pull request by quic-rishinr**
Remove all five Kimi K2.5 QAIC xfail.
Stabilize Kimi K2.5 test initialization and logits parity.
Force full CI while disabling LLM impact-test selection.

### CODE REVIEW

**Comment by quic-rishinr** [comment]
CI-Ready

**Comment by quic-rishinr** [comment]
CI-Ready

**Comment by quic-rishinr** [comment]
CI-Ready
